src/diffusers/loaders/unet.py

```diff
@@ -393,12 +410,12 @@ def save_attn_procs(
         **kwargs,
     ):
         r"""
-        Save an attention processor to a directory so that it can be reloaded using the
+        Save attention processor layers to a directory so that it can be reloaded with the
```

Comment:
here as well

Reply:
Only relevant for custom diffusion at this point: 
https://github.com/huggingface/diffusers/blob/c697f524761abd2314c030221a3ad2f7791eab4e/examples/custom_diffusion/train_custom_diffusion.py#L1268